test/functional/sync_plan/list_test.rb

```diff
@@ -0,0 +1,30 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/sync_plan'
+
+module HammerCLIKatello
+  describe SyncPlan::ListCommand do
+    it 'allows minimal options' do
+      api_expects(:sync_plans, :index) { |p| p['organization_id'] == 1 }
```

Comment:
You can use SyncPlanHelpers here https://github.com/Katello/hammer-cli-katello/blob/master/test/functional/sync_plan/sync_plan_helpers.rb